ntoskrnl/config/cmparse.c

```diff
@@ -502,8 +524,21 @@ CmpDoCreate(IN PHHIVE Hive,
         {
             /* Fail, this shouldn't happen */
             ASSERT(FALSE);
```

Comment:
Now that you've implemented that failure code path, that ASSERT(FALSE) may be removed.